### PR TITLE
Add description to schemes

### PR DESCRIPTION
**Pull request by dingens**
A description is a one-liner describing the key features of the scheme.

I've added descriptions for most schemes, but some are still missing. Please add them if you feel you know a scheme well.

### CODE REVIEW

File: encryption/ntru/ntru.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: NTRU
+description: Based rather closely on the original NTRU from 1998 #TODO
```
Comment (dingens) [inline encryption/ntru/ntru.yaml:2]: I'm unsure here

**Comment by dingens** [comment]
We're still missing descriptions for
* ntru-prime
* three-bears
* FALCON
* LUOV
* mqdss
* Picnic
* SPHINCS+

File: encryption/ntru/ntru.yaml (hunk 1 above)
Comment (franziskuskiefer) [reply encryption/ntru/ntru.yaml:2]: True, but still probabilistic PKE based KEMs. Maybe take the gist from https://eprint.iacr.org/2018/1174.pdf

**Comment by franziskuskiefer** [comment]
A sentence about ntru-prime for @dingens 🙂 I think it's best characterized by this sentence from the spec "NTRU  Prime  uses  a  prime-degree  number  field  with  a  large  Galois  group  and  an  inert modulus, minimizing the number of ring homomorphisms available to the attacker." Or in other words; ntru-prime is a version of NTRU that reduces the attack surface by using a prime-degree number field.

File: encryption/ntru/ntru.yaml (hunk 1 above)
Comment (dingens) [reply encryption/ntru/ntru.yaml]: It seems to me they have switched to correct and deterministic encryption for the round 2 submission, so the stuff from that paper does not apply anymore.

File: encryption/frodo/frodokem-aes/frodokem-aes.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 name: FrodoKEM-AES
+description: using AES for seed expansion. Superior iff hardware acceleration is available.
```
Comment (taudor) [inline encryption/frodo/frodokem-aes/frodokem-aes.yaml:2]: Do we really need a mathematical description here? `iff` may be confusing for some people.

File: encryption/three-bears/three-bears.yaml [hunk 3]
```diff
@@ -1,4 +1,8 @@
 name: Three Bears
+description: >-
+  based on module learning with errors (MLWE) from NewHope and Kyber. The
+  polynomial ring underlying the MLWE module is replaced by integers modulo
+  a generalized Mersenne number.
```
Comment (taudor) [inline encryption/three-bears/three-bears.yaml:5]: @dingens is that ok for you? Or is it too long?

Comment (dingens) [reply encryption/three-bears/three-bears.yaml:5]: how about `based on Module LWE, but the underlying polynomial ring is replaced by integers modulo a generalized Mersenne number`

File: encryption/frodo/frodokem-aes/frodokem-aes.yaml (hunk 2 above)
Comment (dingens) [reply encryption/frodo/frodokem-aes/frodokem-aes.yaml:2]: I thought they might just think it's a typo ;)

**Comment by dingens** [comment]
I'd like to merge this now so we can already use the new schema (in the pending pull requests) and use the data that is there so far. We could then continue tweaking the descriptions in a new branch (and PR). OK?

**Comment by taudor** [comment]
> I'd like to merge this now so we can already use the new schema (in the pending pull requests) and use the data that is there so far. We could then continue tweaking the descriptions in a new branch (and PR). OK?

fine with me

**Pull request merged**